include/mbedtls/sha1.h

```diff
@@ -115,18 +117,25 @@ extern "C" {
 #endif
 
 /**
- * \brief          Output = SHA-1( input buffer )
+ * \brief          This function calculates SHA-1 
```

Comment:
Suggestion: Slightly elaborate, saying "This function calculates the SHA-1 checksum of a buffer."